packages/shopping/public/reset-password-finish.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+  <head>
```

Comment:
Missing mandatory charset:

```suggestion
  <head>
    <meta charset="utf-8">
```

Reply:
This is applied above